usecases/auth/authorization/rbac/model.go

```diff
@@ -126,7 +128,13 @@ func Init(conf rbacconf.Config, policyPath string) (*casbin.SyncedCachedEnforcer
 		if strings.TrimSpace(conf.RootUsers[i]) == "" {
 			continue
 		}
-		if _, err := enforcer.AddRoleForUser(conv.PrefixUserName(conf.RootUsers[i]), conv.PrefixRoleName(authorization.Root)); err != nil {
+
+		// add root role to db as well as OIDC users - we block dynamic users from being created with the same name
+		if _, err := enforcer.AddRoleForUser(conv.UserNameWithTypeFromId(conf.RootUsers[i], models.UserTypesDb), conv.PrefixRoleName(authorization.Root)); err != nil {
+			return nil, fmt.Errorf("add role for user: %w", err)
+		}
+
+		if _, err := enforcer.AddRoleForUser(conv.UserNameWithTypeFromId(conf.RootUsers[i], models.UserTypesOidc), conv.PrefixRoleName(authorization.Root)); err != nil {
```

Comment:
What if the DB doesn't have OIDC configured? Does this affect that at all or are OIDC users just ignored in such a case?

Reply:
They are currently ignored, but I could add validation for this! Ok with a follow up PR?

What happens is that the added role is written to the casbin storage, where it looks like this:
```
g, db:admin-user, role:root
g, oidc:admin-user, role:root
```
Additional entries have no effect, eg the `oidc:admin-user` does not hurt. But I think it makes sense to still validate this